Simple solvers/LL.rb

```diff
@@ -7,45 +7,45 @@
 puts "What is the C value?"
 c = Float(gets)
 
-dis = Float(b**2-4*a*c)
+discriminant = Float(b**2-4*a*c)
  
 puts "User Index"
-puts "* means to multiply(ex. 4*2 = 8)"
+puts "* means to multiply (ex. 4*2 = 8)" 
 puts "** means to the power of (ex. 2**2 = 4)"
 puts "**0.5 means square root (ex. 4**0.5 = 2)"
 puts " / means dividing (ex. 4 / 2 = 2)"
-if dis < 0 
+if discriminant < 0 
     puts "There are no solutions because the discriminant (b**2-4*a*c) is less than 0;"
 
-elsif dis > 0
+elsif discriminant > 0
     puts "There are two solutions because the discriminant (b**2-4*a*c) is greater than 0,"
     puts "Finding solutions using quadratic formula(-b ± (b**2-4*a*c)**0.5) / (2*a)"
     puts "Found! The solutions are rounded to 2 decimal places"
     puts "The solutions are:"
-    xint1 = (-b + (dis)**0.5) /(2*a)
-    xint2 = (-b - (dis)**0.5) /(2*a)
+    xint1 = (-b + (discriminant)**0.5) /(2*a)
+    xint2 = (-b - (discriminant)**0.5) /(2*a)
     puts    xint1.round(2), xint2.round(2)
     puts "Finding solutions using a different method..."
-    b1 = (b / a)
-    c1 = (c / a)
-    m = Float(-b1 / 2)
+    newB = (b / a)
+    newC = (c / a)
+    midpoint = Float(-newB / 2)
     puts "This method will first find the midpoint of the solutions by finding the average of the solutions by dividing B by A, then taking that value and dividing by 2"
-    puts "Found! The midpoint is '#{m.round(2)}'"
-    d = Float((m**2-c1)**0.5)
+    puts "Found! The midpoint is '#{midpoint.round(2)}'"
+    distance = Float((midpoint**2-newC)**0.5)
     puts "This method will now find the distance from the midpoint to each solution by using (m**2-(c / a)**0.5)"
-    puts "Found! The distance is '#{d.round(2)}' to each solution"
-    s1 = m+d
-    s2 = m-d
+    puts "Found! The distance is '#{distance.round(2)}' to each solution"
+    xint1a = midpoint+distance
```

Comment:
Consider an even better name for the variable.  Don't be stingy, make it easy for others (and you in the future) to read your program and maintain it.